packages/fraction-model/src/answer-fraction.jsx

```diff
@@ -19,10 +19,17 @@ export class AnswerFraction extends React.Component {
    * @returns updated answer change object
    */
   onValueChange = (key) => (event) => {
-    const { onAnswerChange, answers } = this.props;
+    let value = parseInt(event.target.value);
```

Comment:
I would suggest adding a sanity check for event.target here to ensure it's defined and has a valid value

Reply:
Updated